Build/Customizations/Projects.props

```diff
@@ -1,7 +1,26 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
-	<ItemGroup>
+  <PropertyGroup>
+    <NormalTestConfiguration Condition="'$(NormalTestConfiguration)' == ''">
+      Win_NET48  + NET48  + NoBrowser + NoDB + Debug   + x86;
```

Comment:
Hmm.. in the build script's own tests we can actually remove all but one entry since it's powershell we're running anyway. The powershell environment is always using NET45 as target framework, regardless of the platform.

Reply:
So the only test configuration besides the local one would be `Win_NET45  + NET45  + NoBrowser + NoDB + Release + x64;`?

Reply:
WIN_NET48 + NET45 or WIN_NET47 + NET45

There's not NET45 docker images and the systems are all updated to newer .NET versions.